Lets the user put girl packs and item packs in more than one place.

UseAfterFree-177 opened the pull request:
closes #18

UseAfterFree-177 committed:
Lets the user put girl packs and item packs in more than one place.

h1262216 commented:
thanks for the patch. Here are a few comments:

I think the string splitting logic could be replaced by `https://www.boost.org/doc/libs/1_74_0/doc/html/boost/algorithm/split.html`.

The tests currently use `:` as separation character, so they should be guarded depending on the OS.

The vector printer for doctest looks like it might be generally useful. Can you make a new util header for doctest related stuff?

UseAfterFree-177 committed:
Now uses boost's split(). vector printer moved to its own header.

UseAfterFree-177 commented:
Updated.

The tests call `do_split_seach_path()`, so they can use whatever separation char they want (and now I've added a few more :smile: ); so they don't need that guard. (`do_split_search_path()` might begin to be a bad name by now, though.)

The vector printer should really be part of doctest itself. Push to those devs?
